robottelo/cli/subscription.py

```diff
@@ -34,9 +35,11 @@ class Subscription(Base):
     def upload(cls, options=None):
         """Upload a subscription manifest."""
         cls.command_sub = 'upload'
+        timeout = 900 if bz_bug_is_open(1339696) else 300
```

Comment:
Picked up random manifest importing task, and it says `"started_at":"2016-05-25 04:07:48 +0300","ended_at":"2016-05-25 04:22:26 +0300",`, it's 14m 38s. Now i'm starting to suspect even 900 seconds may be not always enough :(